internal/pkg/skuba/addons/cilium.go

```diff
@@ -73,6 +75,9 @@ func renderCiliumPreflightTemplate(addonConfiguration AddonConfiguration) string
 	switch {
 	case strings.HasPrefix(ciliumVersion, "1.6"):
 		return ciliumPreflightManifestv16
+	case strings.HasPrefix(ciliumVersion, "1.7"):
+		// ### Do we need a v17 version?
```

Comment:
No, we don't need the preflight deployment when upgrading from 1.6 to 1.7. ;) We can remove this case.

Reply:
removed